h/static/scripts/util/search-text-parser.js

```diff
@@ -122,7 +116,73 @@ function getLozengeValues(queryString) {
   };
 }
 
+/**
+ * Return true if the input query term string has a known named
+ * query term.
+ *
+ * @param {String} queryTerm The query term string.
+ *
+ * @returns {Bool} True if the query term string has a known named
+ * query term and False otherwise.
+ *
+ * @example
+ * // returns false
+ * hasKnownNamedQueryTerm('foo:bar')
+ * @example
+ * // returns true
+ * hasKnownNamedQueryTerm('user:foo')
+ */
+function hasKnownNamedQueryTerm(queryTerm) {
+  const knownNamedQueryTerms = ['user',
+    'url',
+    'group',
+    'tag'];
+
+  var facetName;
+
+  facetName = getLozengeFacetNameAndValue(queryTerm).facetName;
+  if (knownNamedQueryTerms.indexOf(facetName) >= 0) {
```

Comment:
The `if` statement is not required here, you can just write this as `return knownNamedQueryTerms.indexOf(facetName) >= 0)`